src/app/[locale]/_component/sessionActiveComp/__tests__/sessionActiveCompENLocale.test.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
-import { render, screen, fireEvent } from '@testing-library/react';
+import { test, vi } from 'vitest';
 import SessionActiveComp from '../sessionActiveComp';
 import { renderWithProviders } from '@/app/[locale]/_utils/test-utils';
 import { localeFromStorage } from '@/app/[locale]/_utils/common';
 
-jest.mock('../../../_component/accordion/faqDefault.tsx', () => ({
+vi.mock('../../../_component/accordion/faqDefault.tsx', () => ({
   FAQ: () => (
     <div data-testid="mocked-faq">
       Mocked FAQ to avoid SyntaxError: Cannot use import statement outside a module
     </div>
   ),
 }));
```

Comment:
Is this still needed in `vitest` too?

Reply:
Yes it is! It's an issue within `pagopa-editorial-components`.